doc/source/whatsnew/v1.1.0.rst

```diff
@@ -816,6 +816,7 @@ Deprecations
 - :meth:`util.testing.assert_almost_equal` now accepts both relative and absolute
   precision through the ``rtol``, and ``atol`` parameters, thus deprecating the
   ``check_less_precise`` parameter. (:issue:`13357`).
+- Issue warning if value column name already exists when using :meth:`DataFrame.melt` (:issue:`34731`)
```

Comment:
can you start with `DataFrame.melt` (like the other notes above), and also say that this is deprecated and removed in future version.